hoauth2.cabal

```diff
@@ -59,12 +59,16 @@ Library
 
   Build-Depends:
     base                 >= 4     && < 5,
-    aeson                >= 1.0   && < 1.1,
+    aeson                >= 0.11,
     text                 >= 0.11  && < 1.3,
     bytestring           >= 0.9   && < 0.11,
-    http-conduit         >= 2.2   && < 2.3,
+    http-conduit         >= 2.1   && < 2.3,
     http-types           >= 0.9   && < 0.10,
-    unordered-containers >=0.2.5
+    aeson                >= 0.11  && < 1.1,
```

Comment:
It still worked, I think it may have been the `lts-7.18`, but I'm not 100% sure.